site/docs/master/sonobuoy-config.md

```diff
@@ -54,14 +54,45 @@ Filters
    - A Kubernetes [label selector][labelselector] which will be added to every query run.
 
 Limits
- - Options for limiting limits the scope of response when getting logs from pods. These will be passed onto Kubernetes [PodLogOptions][podlogopts]
- - The supported parameters are:
-    - Previous
-    - SinceSeconds
-    - SinceTime
-    - Timestamps
-    - TailLines
-    - LimitBytes
+ - Options for limiting limits the scope of response.
+ - **Limits.PodLogs** limits the scope when getting logs from pods. 
+     - The scope of pods is defined by:
+       ```
+       Namespaces OR SonobuoyNamespace OR FieldSelectors) AND LabelSelector
+       ```
+     - The supported parameters are:
+        - **Namespaces**: string
+            - A regular expression for the targeted namespaces, following the format of Golang Regexp. 
+            - Default is empty string
+            - To get logs from all namespaces use ".*"
+        - **SonobuoyNamespace**: bool
+            - If set to true, take into account the pod logs from Sonobuoy's namespace
+            - Default value is true
+            - Valid inputs: true | false | null
```

Comment:
Why do we want to let the user specify the null value? Is there a reason or it all has to do with the boolean-defaults-to-true issue?

Reply:
I just want to say user can either not provide this field or provide this field with a null value. We can remove it since the null pointer is just a coding implementation, no need to expose to user.